packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -145,10 +146,12 @@ export class SelectionZone extends BaseComponent<ISelectionZoneProps, {}> {
     const target = ev.target as HTMLElement;
     const { selection } = this.props;
     const isToggleModifierPressed = this._isCtrlPressed || this._isMetaPressed;
+    // we should select an item if the focus originates from an arrow down keyboard event
```

Comment:
For general `SelectionZone`, it should be *any* arrow event, right? Really, I'd see if I could scope this behavior quirk to `DetailsList` otherwise.